src/markdown-tests/index.ts

```diff
@@ -72,11 +73,9 @@ export function testPlugin(
       debugEmit(
         applyAllToAllAndGetCounts(
           getOnlyVariant(parse(input, false)),
-          {
-            addWarning: () => {},
-            options: compilationOptionsFromKeywords(args),
-          },
-          ...plugins.map((x) => x.visit),
+          compilationOptionsFromKeywords(args),
+          () => {},
+          ...plugins.map((x) => (typeof x === "function" ? x : x.visit)),
```

Comment:
Is the old `Plugin` (non-`PluginVisitor`) used anymore?

Reply:
Functions that return plugins return the proper `Plugin` type. They could return a function, but then it wouldn't be possible to include the args in the function name. But that wouldn't really matter....
However maybe we will want some more options in the Plugin interface in the future....